src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--block">
```

Comment:
```suggestion
    <div class="stars-container">
```
It is not a modifier of stars block, so let's change the name